src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -294,16 +347,24 @@ public sealed class SystemPropertiesCollection
             /// <summary>
             /// Gets the lock token for the current message.
             /// </summary>
-            /// <remarks>A lock token will only be specified if the message was received using <see cref="ReceiveMode.PeekLock"/></remarks>
+            /// <remarks>
+            ///   The lock token is a reference to the lock that is being held by the broker in <see cref="ReceiveMode.PeekLock"/> mode. 
+            ///   The token can be used to pin the lock permanently through the Deferral API and, with that, take the message out of the 
+            ///   regular delivery state flow. This property is read-only.
```

Comment:
I'm not sure I get this. Deferral and receiving deferred message happens through sequence numbers and not lockToken. So what do you mean by "pinning the lock permanently"?

Reply:
You get a message under a lock and deferral conceptually allows you to hold on to that lock permanently. Defer acts on the lock.